packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -89,15 +92,10 @@ export function resolvePluginConfig(
 		/* prefixes */ ""
 	);
 
-	const configPath = pluginConfig.configPath
-		? path.resolve(root, pluginConfig.configPath)
-		: findWranglerConfig(root);
-
-	if (!configPath) {
-		throw new Error(
-			`Config not found. Have you created a wrangler.json(c) or wrangler.toml file?`
-		);
-	}
+	const configPath = getValidatedWranglerConfigPath(
```

Comment:
Know you didn't change this in this PR but this will help make the code clearer.

```suggestion
	const entryWorkerConfigPath = getValidatedWranglerConfigPath(
```